tools/generate_property_translations.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+#
+# Generates translation strings from gecko property files for lupdate
+# to process
+
+import os, os.path, sys, re, dataclasses, functools, argparse
+
+DEFAULT_PREFIX = "sailfish_components_webview_popups-la"
```

Comment:
Maybe cli argument ... as this is kind of translation key prefix.

Reply:
Is this --prefix or -p somehow insufficient?

Reply:
Yes